src/components/grid-view/GridViewSlot.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../elements/common/variables';
+@import '../../styles/variables';
+
+.bdl-GridViewSlot {
+    flex: 0 1 auto;
+    overflow: hidden;
+    padding: 5px;
+    white-space: normal;
+}
+
+.bdl-GridViewSlot-content {
+    border: 1px solid $bdl-gray-10;
+    box-shadow: 0 1px 5px rgba(50, 50, 50, .1);
+    height: 100%;
+
+    &:hover,
+    &:active,
+    &:focus {
+        border-color: $bdl-box-blue-50;
+    }
+
+    &.bdl-GridViewSlot-content--selected {
```

Comment:
Is it better to keep `&.bdl-GridViewSlot-content--selected` separate from `&:hover, &:active, &:focus` above since `bdl-GridViewSlot-content--selected` is a class, or should I include them all in the same rule?